src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -423,14 +425,14 @@ private void CancellationCallback()
                 {
                     // Remove all watches (inotiy_rm_watch) and clear out the map.
                     // No additional watches will be added after this point.
-                    SysCall(fd => {
-                        foreach (int wd in _wdToPathMap.Keys)
+                    SysCall((fd, runningInstance, _) => {
```

Comment:
@jasonwilliams200OK, I'm not following... why would the file descriptor parameter be renamed from 'fd' to 'watcher'?